diff --git a/test/Microsoft.DotNet.SourceBuild.SmokeTests/SmokeTests.cs b/test/Microsoft.DotNet.SourceBuild.SmokeTests/SmokeTests.cs
index f6a1d96b4..2c732a64b 100644
--- a/test/Microsoft.DotNet.SourceBuild.SmokeTests/SmokeTests.cs
+++ b/test/Microsoft.DotNet.SourceBuild.SmokeTests/SmokeTests.cs
@@ -24,7 +24,7 @@ public SmokeTests(ITestOutputHelper outputHelper)
     [Fact]
     public void SmokeTestsScript()
     {
-        string smokeTestArgs = $"--dotnetDir {Directory.GetParent(DotNetHelper.DotNetPath)} --projectOutput --archiveRestoredPackages --targetRid {Config.TargetRid}";
+        string smokeTestArgs = $"--dotnetDir {Directory.GetParent(DotNetHelper.DotNetPath)} --projectOutput --archiveRestoredPackages --targetRid {Config.TargetRid} --minimal --excludeOnlineTests --excludeOmniSharpTests";
         if (Config.TargetRid.Contains("osx"))
         {
             smokeTestArgs += " --excludeWebHttpsTests";
