diff --git a/src/libraries/Native/Unix/configure.cmake b/src/libraries/Native/Unix/configure.cmake
index e00b35d0f..3f8d70def 100644
--- a/src/libraries/Native/Unix/configure.cmake
+++ b/src/libraries/Native/Unix/configure.cmake
@@ -788,10 +788,15 @@ check_c_source_compiles(
     #include <stdlib.h>
     #include <unistd.h>
     #include <string.h>
+    include <algorithm>
 
+    #define MEM_SIZE 1024
+    #define TEMP_FILE_TEMPLATE \"${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/multiplemaptestXXXXXX\"
     int main(void)
     {
-        return mkstemp(\"abc\");
+      char * fname = (char *)malloc(std::max((size_t)MEM_SIZE, sizeof(TEMP_FILE_TEMPLATE)));
+      strcpy(fname, TEMP_FILE_TEMPLATE);
+      return mkstemp(fname);
     }
     "
     HAVE_MKSTEMP)
